pkg/ccl/kvccl/kvtenantccl/tenant_upgrade_test.go

```diff
@@ -255,6 +255,7 @@ func TestTenantUpgradeFailure(t *testing.T) {
 			TestingKnobs: base.TestingKnobs{
 				JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
 				UpgradeManager: &upgradebase.TestingKnobs{
+					DontUseJobs: true,
```

Comment:
@fqazi this test mocks the cluster versions the server steps through. Because of this, it doesn't run the migration that introduces the `system.job_info` table. Since that migration doesn't run, any migration that starts a job will fail because a write to the jobs info table will fail. Sanity checking that not using jobs for this test is fine?